cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -310,7 +311,15 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
         val resolutionProvider = DefaultResolutionProvider.create(ortResultInput, resolutionsFile)
         val licenseClassifications =
             licenseClassificationsFile.takeIf { it.isFile }?.readValue<LicenseClassifications>().orEmpty()
-        val evaluator = Evaluator(ortResultInput, licenseInfoResolver, resolutionProvider, licenseClassifications)
+        val projectSourceTree = SourceTree.forRemoteRepository(ortResultInput.repository.vcsProcessed)
```

Comment:
This feature should be optional, because downloading the project source code can add significant time to the evaluator run which is wasted if no rule checks any project files. This should be an option in the ORT configuration like `evaluator.downloadProjectSources`, and in my opinion it should default to false.
An alternative solution would be to make the `SourceTree` lazy in the way that it only downloads the source code once it is accessed.

Reply:
>  An alternative solution would be to make the `SourceTree` lazy in the way that it only downloads the source code once it is accessed.

This is actually exactly what I implemented. See `SourceTree.kt` for
```
    val rootDir by lazy {
        getRepositoryRoot()
    }
```

Reply:
Thanks, I missed that.